contracts/contracts/GrantRegistry.sol

```diff
@@ -1,135 +1,124 @@
 pragma solidity ^0.8.5;
 
 contract GrantRegistry {
```

Comment:
We should add a natspec comment for the contract.

Reply:
Done in 756f836a84c1f1b47bf58f12734df5e871fd6b50